src/App.jsx

```diff
@@ -1,40 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabTitle, setSelectedTabTitle] = useState('Tab 1');
+  const [selectedTabContent, setSelectedTabContent] = useState('Some text 1');
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  return (
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
-    </div>
-  </div>
-);
+    <Tabs
+      states={(title, content) => {
+        setSelectedTabTitle(title);
+        setSelectedTabContent(content);
+      }}
+      selectedTitle={selectedTabTitle}
+      selectedContent={selectedTabContent}
+      tabs={tabs}
+      key={tabs.id}
+    />
+  );
+};
```

Comment:
<img width="873" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/9f3cbcc7-304a-42ef-bce3-0c44d5bdfff3">
Pass the onTabSelected callback to the Tabs, it should be called whenever the user selects another tab. (Don't call the callback if the tab was not changed)
try to fix the tests